### DISCUSSION TITLE
Kopieren von mehrzeiligen Listings in das Powershell Fenster

### DISCUSSION THREAD

**Discussion by NdwMan** [TobiasPSP/OReilly#17] [Fragen] [answered]
Hallo da draussen,

irgendwie und irgendwann hat sich bei mir etwas geändert. Ich kann mehrzeilige
Listings die ich mir mit "L x.xx" geholt habe, nicht mehr in das Powershell 7.2.5
Fenster einfügen (STRG-V). Bzw. ich kann es schon, aber es wird quasi jede
Zeile, auch abgeschnittene Zeilen sofort ausgeführt. Das war doch früher 
anders, hat MS da was geändert?

Danke und Gruss

**Comment by TobiasPSP** [comment, marked as answer]
Das Einfügen von mehrzeiligen Listings in eine Konsole ist technikbedingt immer schon etwas schwierig gewesen. Ein Workaround wäre, den einzufügenden Code in geschweifte Klammern zu stellen und einen Punkt davorzusetzen, also zum Beispiel so:

```powershell
. {
'Erste Zeile'
$a = Read-Host "Etwas eingeben"
"Ausgabe: $a"
}
```
